editor/src/components/canvas/controls/yoga-control.tsx

```diff
@@ -67,11 +69,44 @@ class YogaResizeControl extends React.Component<YogaResizeControlProps> {
       // TODO check me
       return null
     }
-
+    let labels: {
+      vertical: 'flexBasis' | 'FlexCrossBasis'
```

Comment:
Why are we using lowercase for `flexBasis` but not for `FlexCrossBasis`?

Reply:
Because that's how the keys are defined in the map which backs up these fields.

Reply:
this was from the times when we still had the `layout` property, which included support for `FlexCrossBasis`. We should probably eradicate that from the map